### PR TITLE
made edits

### PR DESCRIPTION
**Pull request by priyaayyar**
- Added note for oas3 support.
- Made edits for content that was inconsistent with the UI.

### CODE REVIEW

File: modules/ROOT/pages/create-instance-task.adoc [hunk 1]
```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
 
 After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-To create a API instance:
+To create an API instance:
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+. Select *API Administration* > *Manage API* > *Create a new API*.
```
Comment (sbehrends) [inline modules/ROOT/pages/create-instance-task.adoc:14]: Here you are changing the flow from "Manage API from Exchange" to "Create New API" is this change intended?

- "Manage API from Exchange": User is able to search and select an asset published in Exchange
- "Create New API": Will allow user to upload an asset to Exchange and Manage such assets in one step.

File: modules/ROOT/pages/create-instance-task.adoc [hunk 2]
```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
...
 +
-For example, to set up version 1.0 of an API named `jsonplaceholder`:
+For example, set up version 1.0 of an API named `jsonplaceholder`:
```
Comment (sbehrends) [inline modules/ROOT/pages/create-instance-task.adoc:16]: Would this asset be already published in Exchange or is the user uploading the asset?

File: modules/ROOT/pages/create-instance-task.adoc [hunk 3]
```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
...
 image::manage-exchange-end-task.png[]
 +
 . From *API Name*, select the API (`jsonplaceholder`).
-. From *API Version*, select version (`1.0`).
-. From *Asset Version*, select the Exchange asset version.
+. From *Asset types*, select the type of API specification that you want to create.
++
+[Note] OAS 3.0 specifications are currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage using API Manager.
++
+. If you are creating an API type other than HTTP, upload the specification file in the *File upload* field.
```
Comment (sbehrends) [inline modules/ROOT/pages/create-instance-task.adoc:25]: Upload of Asset is only required if they are using the "Create New API" flow. If they are managing from Exchange, user is searching on their Exchange Assets.

File: modules/ROOT/pages/create-instance-task.adoc [hunk 4]
```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
...
+. Click *Continue*.
+// . From *API Version*, select version (`1.0`).
+// . From *Asset Version*, select the Exchange asset version.
```
Comment (sbehrends) [inline modules/ROOT/pages/create-instance-task.adoc:28]: What is this `// .` format for?
Is it a comment of something to be removed?

File: modules/ROOT/pages/create-instance-task.adoc [hunk 5]
```diff
@@ -9,20 +9,29 @@ You can create one instance of an API that serves as a proxy. You can create ano
...
 . From *Managing Type*, choose *Basic Endpoint* or *Endpoint with Proxy*. 
 +
-For this example, select *Endpoint With Proxy*.
+* Basic Endpoint: Configure a basic endpoint if you create your API outside API Manager, but want to apply policies or otherwise manage the API using API Manager. For example, if you create the API using Mule, you do not need a proxy.
```
Comment (sbehrends) [inline modules/ROOT/pages/create-instance-task.adoc:31]: Here is a description of "Basic Endpoint" vs "Endpoint with Proxy"
https://docs.mulesoft.com/api-manager/2.x/configure-api-task

File: modules/ROOT/pages/create-instance-task.adoc [hunk 6]
```diff
@@ -35,12 +44,18 @@ image::create-instance-endpoint-midp.png[65%,65%, Create instances with multiple
...
-. In Advanced Options, optionally assign an instance label for the proxy, or accept the default label, and set a response timeout.
+.  Optionally, specify additional parameters in *Advanced Options*:  
+* Assign an instance label to the basic endpoint or proxy.
++
+In case you create multiple instances, labeling them with a user-friendly name is recommended.
+* Specify a response timeout.
+* If you choose *Endpoint with Proxy* in the previous step, specify a scheme (HTTP or HTTPS) and a port, or accept the defaults.
+. If you choose *Hybrid* as the proxy deployment target, check Reference User Domain to use an API Gateway domain.
++
+You must install the API gateway domain in Mule 3.8 and later.
```
Comment (sbehrends) [inline modules/ROOT/pages/create-instance-task.adoc:55]: This is only if using Reference User Domain

File: modules/ROOT/pages/manage-exchange-api-task.adoc [hunk 7]
```diff
@@ -3,49 +3,42 @@ ifndef::env-site,env-github[]
...
-+
-A link to the RAML/OAS specification for viewing appears.
+. In *API Name*, select the API you want to manage from the list of available APIs in Anypoint Exchange (Exchange).
 +
-* In Managing Type, choose Basic Endpoint or Endpoint with Proxy.
+The *Asset type*, *API version*, and *Asset version*, fields are auto-populated. A link to the RAML or OAS specification for viewing appears.
 +
-Set up a basic endpoint if you create your API outside API Manager, but you want to apply policies or otherwise manage the API using API Manager. For example, if you create the API using Mule Runtime, you don't need a proxy.
+[Note] OAS 3.0 specifications are currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage using API Manager.
+. In *Managing Type*, choose *Basic Endpoint* or *Endpoint with Proxy*:
```
Comment (sbehrends) [inline modules/ROOT/pages/manage-exchange-api-task.adoc:21]: Same feedback as above.

**Review by sbehrends — CHANGES_REQUESTED** [review]
Left some comments regarding some wording changes etc.
OAS3 Notices looks good.

**Comment by priyaayyar** [comment]
Created a separate PR for this topic.

**Pull request closed without merging**